apps/web/src/pages/changes/PromoteChangesPage.tsx

```diff
@@ -29,6 +29,9 @@ export function PromoteChangesPage() {
       queryClient.refetchQueries([QueryKeys.changesCount]);
       successMessage('All changes were promoted');
     },
+    onError: (err: any) => {
+      errorMessage(err?.message);
```

Comment:
Maybe add a default error message? Just in case err.message is empty?

Reply:
`Something went wrong!` ?? 😅

Reply:
Maybe - 'Could not promote changes' ? Or 'Something went wrong! Failed to promote changes' ? Wdyt?

Reply:
'Something went wrong! Failed to promote changes'  seems better.